charts/app-config/values-integration.yaml

```diff
@@ -1415,6 +1415,21 @@ govukApplications:
             secretKeyRef:
               name: signon-app-chat
               key: oauth_secret
+        - name: SMART_SURVEY_URL
+          valueFrom:
+            secretKeyRef:
+              name: govuk-chat-smart-survey
+              key: url
```

Comment:
Does this actually need to be a secret? I'd wonder if we'd just have this set-up in our code base and not need an env var for it? is it: https://api.smartsurvey.io/ ?

Reply:
it has the id of our survey in it

Reply:
Right - hmm that seems to be a bit unusual as we'd be storing exactly one API endpoint specifically. I don't think we should do that.

I think we should have the survey id separate though so we can use it to form a url. I don't think it actually needs to be secret though it would be in [the URL of our survey](https://surveys.publishing.service.gov.uk/s/govuk-chat-beta/?user=b1f782ef-b3f6-4164-9689-9ecca83333c6) if we hadn't set a custom slug and it is available in the HTML source code of the page (1561650) - so I'd expect we just want it in our rails config unless we want to test with other surveys and thus have it as an env var.